src/frontend/screens/Observation/TrackList.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import ChainIcon from '../../images/Chain.svg';
+import {Track} from '@comapeo/schema';
+import {useNavigationFromRoot} from '../../hooks/useNavigationWithTypes.ts';
+import {defineMessages, useIntl} from 'react-intl';
+import {Accordian} from '../../sharedComponents/Accordian.tsx';
+import {HeaderText} from '../../sharedComponents/Text/HeaderText.tsx';
+import {TrackListItem} from '../ObservationsList/TrackListItem.tsx';
+
+interface TrackListProps {
+  track: Track;
+}
+
+const m = defineMessages({
+  track: {
+    id: 'screens.Observation.TrackList.track',
+    defaultMessage: 'Track',
+  },
+});
+
+export function TrackList({track}: TrackListProps) {
```

Comment:
nit: This isn't really a list even though I get why it contains a tracklistitem. Can it be called something more semantic? Like TrackLink?